tests/modules/last/lastal/test.yml

```diff
@@ -1,45 +1,21 @@
 - name: last lastal test_last_lastal_with_dummy_param_file
-  command: nextflow run ./tests/modules/last/lastal -entry test_last_lastal_with_dummy_param_file -c ./tests/config/nextflow.config -c ./tests/modules/last/lastal/nextflow.config
+  command: nextflow run tests/modules/last/lastal -entry test_last_lastal_with_dummy_param_file -c tests/config/nextflow.config
   tags:
     - last
     - last/lastal
   files:
     - path: output/last/contigs.genome.maf.gz
-      md5sum: 670f4fa1a94b23690cdb6fc603813c75
-    - path: output/untar/lastdb/genome.bck
-      md5sum: 5519879b9b6c4d1fc508da7f17f88f2e
-    - path: output/untar/lastdb/genome.des
-      md5sum: 3a9ea6d336e113a74d7fdca5e7b623fc
-    - path: output/untar/lastdb/genome.prj
-      md5sum: 489715f14b0fea6273822696e72357f9
-    - path: output/untar/lastdb/genome.sds
-      md5sum: 2cd381f4f8a9c52cfcd323a2863eccb2
-    - path: output/untar/lastdb/genome.ssp
-      md5sum: 4137fb6fe9df2b3d78d5b960390aac7b
-    - path: output/untar/lastdb/genome.suf
-      md5sum: 1895efa8653e8e9bd3605cff0408ed33
-    - path: output/untar/lastdb/genome.tis
-      md5sum: b7c40f06b1309dc6f37849eeb86dfd22
+      md5sum: 90a9b08b228ee25dfe22f78d3dc72208
```

Comment:
Is it correct these are being deleted entirely?

Reply:
Yes, it is the output of the `untar` module, which is not the target of the tests.